vignettes/customise.Rmd

```diff
@@ -21,6 +21,8 @@ We'll start by discussing two techniques that only require tweaks to your `_pkgd
 We'll then discuss how to add additional HTML and other files.
 Next, we'll discuss how to give multiple sites the same style using a package, then finish up with some workflow advice.
 
+In terms of your `_pkgdown.yml`, this vignette focusses on the most important fields nested under `template` and `navbar`. To learn more about customising other aspect of the site, see the documentation for the indiviudal functions like `build_reference()`, `build_articles()`,  `build_home()`, and `init_site()`. To learn about less important fields nested under `template`, see `build_site()`.
```

Comment:
```suggestion
In terms of your `_pkgdown.yml`, this vignette focusses on the most important fields nested under `template` and `navbar`. To learn more about customising other aspect of the site, see the documentation for the indiviudal functions like `build_reference()`, `build_articles()`,  `build_home()`, `build_redirects()`, and `init_site()`. To learn about less important fields nested under `template`, see `build_site()`.
```

Since the `redirects:` field is now documented in `build_redirects` and some things like language? I think that is good thing to mention.

Language  `lang:` is only documented in the intro vignette, maybe worth documenting this in `build_site()` too? I couldn't find cross-references that link to it elsewhere in the documentation.

Reply:
I think `lang` should probably go in a new vignette, since we also need to document how to contribute translations. #2605